### ISSUE TITLE
Preprocessing padding

### ISSUE THREAD

**Issue by ggoupy** [ShahriarRezghi/Spyker#4]
Parameter to choose the padding method in preprocessing methods (e.g. DoG, LoG, ..).

For real images, it creates high activation on the edges.

**Comment by ShahriarRezghi** [comment]
That might be because you are padding the image with zeros, and this creates a high contrast near the edges, and results in high activation. Is this correct?

**Comment by ggoupy** [comment]
Exactly. And as far as I know, you can not choose other padding methods (e.g. "mirror", "edge") with the `spyker.pad` or the `spyker.DoG`.

**Comment by ShahriarRezghi** [comment]
The back-end libraries used (cuDNN, etc) don't support such padding methods as far as I know, and I do not plan to add such features for now. You can use other tools to pad the input before passing it to `spyker.DoG`.

**Comment by ggoupy** [comment]
Yes of course, it can still be done before the preprocessing step. 
Just one might be aware that the padding of the DoG method can create weird processing with some images.

**Comment by jkyle1234** [comment]
I'm having a problem import the pybind11 module. I was able to compile it and created spyker.pyd However I am unable to successfully import it ie call import spyker (on windows). Any suggestions with that?

 DLL load failed while importing spyker: The specified module could not be found.

**Comment by ShahriarRezghi** [comment]
@jkyle1234 the problem you are having is not related to this issue. Please open another issue, and also describe your problem with a lot more detail (like the version of dependencies you are using, build configuration, sample code, etc).
